docs/quick_tutorial/authorization/setup.py

```diff
@@ -2,11 +2,12 @@
 
 requires = [
     'bcrypt',
-    'pyramid',
+    'pyramid[testing]',
```

Comment:
This should be removed, external apps should not depend on "whatever pyramid uses to run its own test suite". If you want to define some testing dependencies for this authorization app then normally we'd do this by specifying a "testing" extras requirement. See how it is done in the starter cookiecutter for an example. Specifically there is a `tests_require` defined in `setup.py` and instructions to run `pip install -e ".[testing]"` to install the requirements.

https://github.com/Pylons/pyramid-cookiecutter-starter/tree/latest/%7B%7Bcookiecutter.repo_name%7D%7D